README.md

```diff
@@ -1,5 +1,5 @@
-# quito_1
+# flutter_quito_app
 
 This project is a Flutter application for the 2019 Calico Challenge Quito project. 
 
-This application currently includes the login screen for the app as well as a chat interface for registered and authorized users and screens to begin creating events for the Quito app abd send them to the server
\ No newline at end of file
+This application currently includes the login screen for the app as well as a chat interface for registered and authorized users and screens to begin creating events for the Quito app abd send them to the server
```

Comment:
there is a typographical error in the last line. It should be "... the Quito app and send them to the server" instead of "... the Quito app abd send them to the server"

```suggestion
This application currently includes the login screen for the app as well as a chat interface for registered and authorized users and screens to begin creating events for the Quito app and send them to the server
```